modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="AdminNetworkPolicy"_{context}"]
+= AdminNetworkPolicy
+
+`AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `AdminNetworkPolicy` to secure your cluster in the following ways:
```

Comment:
```suggestion
`AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definition (CRD) that is maintained and shipped by sig-network-policy-api working group <link to https://github.com/kubernetes-sigs/network-policy-api>. As an {product-title} administrator, you can use `AdminNetworkPolicy` to secure your cluster's networking in the following ways:
```

Reply:
We have links to sig docs in other spaces so I'm just adding this to the Additional resources section.

Reply:
ack I didn't find it anywhere, must have missed the link to the KEP. Could you please tell me where?

Reply:
https://docs.openshift.com/container-platform/4.14/machine_management/capi-machine-management.html